lib/iris/cube.py

```diff
@@ -2302,7 +2306,8 @@ def _as_list_of_coords(self, names_or_coords):
         Convert a name, coord, or list of names/coords to a list of coords.
         """
         # If not iterable, convert to list of a single item
```

Comment:
This comment is a little misleading with py3k strings having gained an `__iter__` method...